### PR TITLE
Create facilities

### PR DESCRIPTION
**Pull request by jgonz11**

### CODE REVIEW

**Comment by edumorlom** [comment]
please fix conflicts

File: src/Components/Homepage.jsx [hunk 1]
```diff
@@ -8,41 +20,66 @@ import appStyles from './AppStyles';
 import CancelFilterButton from './Button';
 import {getRef} from '../Firebase';
 import filterImage from '../../assets/delete-filter.png';
+>>>>>>> acb8192fc8d6d2edaefad2e5a63894c8e7f3a971
```
Comment (edumorlom) [inline src/Components/Homepage.jsx:23]: They were fixed correctly :(

**Review by edumorlom — CHANGES_REQUESTED** [review]

File: src/en_US.json [hunk 2]
```diff
@@ -136,6 +136,9 @@
   "Yearly Exam": "Yearly Exam",
   "Emergency Testing": "Emergency Testing",
   "Miles": "Miles",
+<<<<<<< HEAD
+  "facilities": "Facilities"
```
Comment (jgonz11) [inline src/en_US.json:140]: facilities

**Review by jgonz11 — COMMENTED** [review]
All files have been reviewed and fixed all conflicts

**Review by edumorlom — APPROVED** [review]

**Comment by edumorlom** [comment]
Seems to work! Thanks!

**Pull request merged**